lib/markdown2.py

```diff
@@ -1029,6 +1029,9 @@ def _run_block_gamut(self, text):
         return text
 
     def _pyshell_block_sub(self, match):
+        if "fenced-code-blocks" in self.extras:
+            dedented = _dedent(match.group(0))
+            return self._do_fenced_code_blocks("```pycon\n" + dedented + "```\n")
```

Comment:
This `pycon` stood out to me. Is it supposed to be `python` or maybe `pydoc`? Wasn't sure, seemed odd. Thanks!

Reply:
Good that you are checking! 😃  `pycon` is the name of the pygments' Python Console Lexer (which supports the >>> syntax): https://pygments.org/docs/lexers/#pygments.lexers.python.PythonConsoleLexer

Reply:
Ah, got it, thank you